src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyTotalAmount = 0;
+        int buyTotalAmount = 0;
+        for (String line: readFromFile(fromFileName).split("\\r?\\n")) {
+            String[] row = line.split(",");
+            if (row[0].equals("supply")) {
+                supplyTotalAmount += Integer.valueOf(row[1]);
+            } else {
+                buyTotalAmount += Integer.valueOf(row[1]);
+            }
+        }
+        StringBuilder result = new StringBuilder();
+        result.append("supply," + supplyTotalAmount + System.lineSeparator());
+        result.append("buy," + buyTotalAmount + System.lineSeparator());
+        result.append("result," + (supplyTotalAmount - buyTotalAmount));
```

Comment:
you can call append() several times in a row
let's avoid concatenation at all

result.append("supply").append(supplyTotalAmount).append(System.lineSeparator()).append(.......)